app/Filament/Resources/ApiTokenResource.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace App\Filament\Resources;
+
+use App\Filament\Resources\ApiTokenResource\Pages;
+use Filament\Forms\Components\Card;
+use Filament\Forms\Components\CheckboxList;
+use Filament\Forms\Components\DateTimePicker;
+use Filament\Forms\Components\TextInput;
+use Filament\Forms\Form;
+use Filament\Resources\Resource;
+use Filament\Tables\Actions\ActionGroup;
+use Filament\Tables\Actions\DeleteAction;
+use Filament\Tables\Actions\DeleteBulkAction;
+use Filament\Tables\Actions\EditAction;
+use Filament\Tables\Columns\IconColumn;
+use Filament\Tables\Columns\TextColumn;
+use Filament\Tables\Filters\MultiSelectFilter;
+use Filament\Tables\Filters\TernaryFilter;
+use Filament\Tables\Table;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Support\Facades\Auth;
+use Laravel\Sanctum\PersonalAccessToken;
+
+class ApiTokenResource extends Resource
+{
+    protected static ?string $model = PersonalAccessToken::class;
+
+    protected static ?string $navigationIcon = 'tabler-api';
+
+    protected static ?string $navigationGroup = 'Settings';
+
+    protected static ?string $label = 'API Token';
+
+    protected static ?string $pluralLabel = 'API Tokens';
+
+    public static function getTokenFormSchema(): array
+    {
+        return [
+            Card::make()->schema([
```

Comment:
`Card` was deprecated, let's remove that wrapper. If you need to format the schema you can use use grid layout instead: https://filamentphp.com/docs/3.x/forms/layout/grid